### PR TITLE
Iterate readme

### PR DESCRIPTION
**Pull request by benthorner**
https://trello.com/c/MIM5hTAm/123-setup-govuk-connect-as-a-separate-repo-and-tool-for-ssh

Please see the commits for more details.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # GOV.UK Connect
 
-This is a command line tool to help you connect to GOV.UK's infrastructure.
+This is a command line tool to help you connect to GOV.UK's infrastructure e.g.
+
+```bash
+govuk-connect ssh -e integration frontend
```
Comment (issyl0) [inline README.md]:
```suggestion
gds govuk connect ssh -e integration frontend
```

One tool, then people don't get confused between whether they're SSHing or AWSing? Is it worth specifying the two commands?

File: README.md [hunk 2]
```diff
@@ -1,11 +1,23 @@
...
+```
 
 ## Installation
 
 To install:
 
 ```bash
-gem install govuk-connect
+sudo gem install govuk-connect
```
Comment (issyl0) [inline README.md]: I feel like there should be a Better Way.

Comment (benthorner) [reply README.md]: There is: clone the repo, add it to the path. Now where have I seen that before 🤔

File: README.md (hunk 1 above)
Comment (benthorner) [reply README.md]: I think we should keep the guidance here self-contained, especially if someone doesn't have the GDS CLI installed (e.g. if they're not using AWS yet). Perhaps we can add another bit of guidance that talks about installing GDS CLI and subsequent usage?

File: README.md (hunk 2 above)
Comment (issyl0) [reply README.md]: `brew install` is the logical step ;-)

File: README.md (hunk 1 above)
Comment (issyl0) [reply README.md]: Fair enough, yeah. We can work on that guidance in the docs card in dev tools!

**Pull request merged**